hack/push_updates.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+#
+# Push updates that need to be vendored in other repositories.
+
+if ! which hub 2>/dev/null 1>&2; then
+    echo "This tool requires the hub command line app." 1>&2
```

Comment:
Is it possible just to use `git` tool?.. Don't know all the details, but from visual inspection, it is possible to replicate all commandls with `git clone` and `git remote add ...`

Reply:
git can't create the fork if you don't already have it